control/control.go

```diff
@@ -0,0 +1,104 @@
+package control
+
+import (
+	"encoding/json"
+	"errors"
+	"net"
+	"net/http"
+	"os"
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	SOCKET_TYPE = "unix"
```

Comment:
I hear yah. It's not a biggie and helps contributions from the community. But still not a fan. Constants should easily read in an API and stand out in the middle of an implementation. Helps narrow down on what defaults are versus what internal data structures are representing. Maybe it's the long lost C programmer in me.